lighthouse-core/gather/computed/critical-request-chains.js

```diff
@@ -34,12 +34,13 @@ class CriticalRequestChains extends ComputedArtifact {
    */
   isCritical(request) {
     // XHRs are fetched at High priority, but we exclude them, as they are unlikely to be critical
```

Comment:
the comments relate to the `nonCriticalResourceTypes` so they shoudl probably be right on top of it.